tests/integration/nm/profile_test.py

```diff
@@ -399,6 +410,15 @@ def test_linux_bridge_with_port_holding_two_profiles(eth1_with_two_profiles):
 
 
 @pytest.mark.tier1
+@pytest.mark.xfail(
+    is_k8s(),
+    reason=(
+        "Requires adjusts for k8s. Ref:"
+        "https://github.com/nmstate/nmstate/issues/1579"
+    ),
+    raises=libnmstate.error.NmstateValueError,
```

Comment:
Use `NmstateValueError` directly, it is imported.

Reply:
Done